doc/sailfish-webview-webview.qdoc

```diff
@@ -141,8 +274,35 @@
 */
 
 /*!
-  \internal
+  \qmlsignal WebView::linkClicked(string url)
+  \brief This signal is emitted when the user selects a link on the page.
 
+  When emitted \a url will be set to href selected.
+*/
+
+/*!
+  \qmlsignal WebView::loadRedirect()
+  \brief Emitted if load() resulted in a redirect
+*/
+
+/*!
+  \qmlsignal WebView::securityChanged(string status, int state)
+  \brief Emitted if the security status of the loaded URL changes
+  \sa security
+*/
+
+/*!
+  \qmlsignal WebView::viewDestroyed()
+  \brief Emitted when the view is destroyed
+*/
+
+/*!
```

Comment:
Let's leave this one out as well. => internal

Reply:
done